accepted/2023/nuget-config-apikeys-unencrypted.md

```diff
@@ -0,0 +1,124 @@
+# Allow clear text API keys in nuget.config
+
+- Andy Zivkovic ([zivkan](https://github.com/zivkan))
+- Start Date (2022-12-15)
+- GitHub Issue (https://github.com/NuGet/Home/issues/6437)
+- GitHub PR (not yet implemented)
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+
+Enable unencrypted API keys in `nuget.config` files, so that they can be used on platforms other than Windows.
+
+## Motivation 
```

Comment:
Last week I had an idea which probably means that the benefit of allowing unencrypted API keys in nuget.config becomes close to zero. It would still be useful for customers who use Mac and Linux (until https://github.com/NuGet/Home/issues/1851 is implemented) and push packages with APIs from their dev box.  But most of the feedback we got was from customers pushing in CI scripts and docker containers, so I think this will satisfy most people:

* https://github.com/NuGet/Home/issues/12539